data/boosters/znr-collector.yaml

```diff
@@ -15,15 +15,15 @@ pack:
     chance: 1 # "about 1 in every 6 Collector Boosters"
 sheets:
   extended_rare_mythic:
-    filter: "e:znr frame:extendedart"
-    use: rare_mythic
+    rawquery: "(e:znr OR e:znc) frame:extendedart"
+    count: 77
   foil_rare_mythic:
     foil: true
     use: rare_mythic
   foil_rare_mythic_showcase:
     foil: true
-    filter: "e:znr promo:boosterfun -is:foilonly is:paper"
-    use: rare_mythic
+    rawquery: "(e:znr OR e:znc) promo:boosterfun (r:m OR r:r)"
```

Comment:
This is:paper filter was dealing with [those two Arena cards](https://mtg.wtf/card?q=%28e%3Aznr+OR+e%3Aznc%29+promo%3Aboosterfun+%28r%3Am+OR+r%3Ar%29+%2B%2B+-is%3Apaper), so it's probably needed?

The query returns 104 cards now.

Reply:
good catch! i'll amend